src/velero/zarf.yaml

```diff
@@ -1,11 +1,42 @@
 kind: ZarfPackageConfig
 metadata:
   name: uds-core-velero
-  description: "Example description"
+  description: "UDS Core Velero"
+  url: https://velero.io/
 
 components:
-  - name: test
-    actions:
-      onCreate:
-        after:
-          - cmd: echo "What can Doug do for you?"
+  - name: velero
+    required: true
+    only:
+      flavor: upstream
+    import:
+      path: common
+    charts:
+      - name: velero
+        valuesFiles:
+          - values/upstream-values.yaml
+    images:
+      - velero/velero:v1.13.0
+      - velero/velero-plugin-for-csi:v0.7.0
+      - velero/velero-plugin-for-aws:v1.9.0
+      - velero/velero-plugin-for-microsoft-azure:v1.8.2
+      - docker.io/bitnami/kubectl:1.28.6
+
+  - name: velero
+    required: true
+    only:
+      flavor: registry1
+    import:
+      path: common
+    charts:
+      - name: velero
+        valuesFiles:
+          - values/registry1-values.yaml
+    images:
+      # The kubectl image used by default in the repo1 chart does not support the runtime command of the upstream chart (`which` has been removed in the IB image) -- suggest replacing with an alternative kubectl image
+      # - registry1.dso.mil/ironbank/opensource/kubernetes/kubectl:v1.28.6
+      - registry1.dso.mil/ironbank/opensource/kubernetes-1.21/kubectl:v1.21.14
```

Comment:
Placeholder comment - wanted to revisit the ironbank images and see if I could find any that provide a newer/supported kubectl with `which`.

Reply:
Haven't identified any drop in replacements - I opened an IB issue [here](https://repo1.dso.mil/dsop/opensource/kubernetes/1.28/kubectl/-/issues/20) to request adding `which`. I'm tentatively okay with this as a shortterm. My fears:
- velero could use some newer kubectl args/commands that don't work with 1.21 (not super likely)
- security posture/CVEs with this image - it will never get updated so we can't adhere to any CVE SLAs/bugfixes

It would be ideal if we could just disable this job and install the CRDs separately, similar to how we handle with some of the other pieces of uds-core...but not sure that is cleaner in this case since they don't publish CRDs in a separate chart.

Reply:
Agreed. I say we leave it for now, but track that issue and update if/when an alternative is available. In the meantime, if it becomes a CVE issue, we can fall back to adding each individual file as a manifest, [like we have done elsewhere](https://github.com/defenseunicorns/uds-package-postgres-operator/blob/main/zarf.yaml#L10).